public/components/navbar.html

```diff
@@ -1,13 +1,12 @@
-<nav class="navbar navbar-expand border border-dark">
+<nav
+  class="navbar navbar-expand-lg bg-body-tertiary text-light"
+  style="background-color: #2a2f35 !important"
+>
   <div class="container-fluid">
-    <div class="collapse navbar-collapse">
-      <ul class="navbar-nav me-auto">
-        <li class="nav-item nav-link">
-          <a href="/index.html"> Home </a>
-        </li>
-      </ul>
-      <div class="position-absolute end-0 d-flex align-items-center mx-5">
-        <a href="/cart.html" style="text-decoration: underline"> Cart </a>
+    <a class="navbar-brand text-light" href="./index.html">LOGO HERE</a>
+    <div class="" id="navbarNav">
+      <div class="ms-auto text-light">
+        <i class="bi bi-cart2 text-light fs-2"></i>
```

Comment:
could we make this a link that redirects to `/cart.html`?

Reply:
Sure